src/style.css

```diff
@@ -0,0 +1,146 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
```

Comment:
If you used a numeric value for the weight above you should use it everywhere. For instance, `font-weight: 700`.